Client/metrics.yaml

```diff
@@ -670,6 +670,24 @@ shopping:
       - fx-ios-data-stewards@mozilla.com
     expires: "2024-06-01"
     unit: counter of product pages
+
+  ads_exposure:
+    type: event
+    description: |
+      On a supported product page, the review checker showed analysis, and
+      the ads exposure pref was enabled, or review checker ads were enabled,
```

Comment:
We only have one pref so this comment should be updated to reflect that.

Reply:
Actually, I think with `review checker ads were enabled` you mean that the Nimbus feature is enabled, correct? In that case ignore my previous comment.

Reply:
Yes, it's the same description as Desktop.